jmdaemon/jmdaemon/daemon_protocol.py

```diff
@@ -212,6 +212,12 @@ def on_JM_MSGSIGNATURE_VERIFY(self, verif_result, nick, fullmsg, hostid):
             self.mcc.on_verified_privmsg(nick, fullmsg, hostid)
         return {'accepted': True}
 
+    @JMShutdown.responder
+    def on_JM_SHUTDOWN(self):
+        print("reached on shutdown in jmdaemonserverprotocol")
```

Comment:
Use the logger functions instead of print?

Reply:
Yeah debug leftover, mistake, sorry.